packages/react-components/src/templates/TeamProfileHeader.tsx

```diff
@@ -247,7 +249,9 @@ const TeamProfileHeader: React.FC<TeamProfileHeaderProps> = ({
         {tools && (
           <TabLink href={route.workspace({}).$}>Team Workspace</TabLink>
         )}
-        <TabLink href={route.outputs({}).$}>Team Outputs</TabLink>
+        <TabLink href={route.outputs({}).$}>
+          Shared Outputs ({sharedOutputsCount})
```

Comment:
Shared Outputs Count should default to zero as the value is optional?

Reply:
yes, you're right! changing it now